specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2019-05-13/bms.json

```diff
@@ -0,0 +1,3352 @@
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "protected": {
+          "description": "To check if backup item is disk protected.",
+          "type": "boolean"
+        },
+        "isPresentOnCloud": {
+          "description": "To check if backup item is cloud protected.",
+          "type": "boolean"
+        },
+        "lastBackupStatus": {
+          "description": "Last backup status information on backup item.",
+          "type": "string"
+        },
+        "lastRefreshedAt": {
+          "format": "date-time",
+          "description": "Last refresh time on backup item.",
+          "type": "string"
+        },
+        "oldestRecoveryPoint": {
+          "format": "date-time",
+          "description": "Oldest cloud recovery point time.",
+          "type": "string"
+        },
+        "recoveryPointCount": {
+          "format": "int32",
+          "description": "cloud recovery point count.",
+          "type": "integer"
+        },
+        "onPremiseOldestRecoveryPoint": {
+          "format": "date-time",
+          "description": "Oldest disk recovery point time.",
+          "type": "string"
+        },
+        "onPremiseLatestRecoveryPoint": {
+          "format": "date-time",
+          "description": "latest disk recovery point time.",
+          "type": "string"
+        },
+        "onPremiseRecoveryPointCount": {
+          "format": "int32",
+          "description": "disk recovery point count.",
+          "type": "integer"
+        },
+        "isCollocated": {
+          "description": "To check if backup item is collocated.",
+          "type": "boolean"
+        },
+        "protectionGroupName": {
+          "description": "Protection group name of the backup item.",
+          "type": "string"
+        },
+        "diskStorageUsedInBytes": {
+          "description": "Used Disk storage in bytes.",
+          "type": "string"
+        },
+        "totalDiskStorageSizeInBytes": {
+          "description": "total Disk storage in bytes.",
+          "type": "string"
+        }
+      }
+    },
+    "EncryptionDetails": {
+      "description": "Details needed if the VM was encrypted at the time of backup.",
+      "type": "object",
+      "properties": {
+        "encryptionEnabled": {
+          "description": "Identifies whether this backup copy represents an encrypted VM at the time of backup.",
+          "type": "boolean"
+        },
+        "kekUrl": {
+          "description": "Key Url.",
+          "type": "string"
+        },
+        "secretKeyUrl": {
+          "description": "Secret Url.",
+          "type": "string"
+        },
+        "kekVaultId": {
+          "description": "ID of Key Vault where KEK is stored.",
+          "type": "string"
+        },
+        "secretKeyVaultId": {
+          "description": "ID of Key Vault where Secret is stored.",
+          "type": "st
```

Comment:
Can this be modeled as an enum? This is a flag that doesn't follow your established 'is___' pattern.

Reply:
Removed. This feature is not applicable to SDK users.